packages/dd-trace/src/appsec/standalone.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const { channel } = require('dc-polyfill')
+const startCh = channel('dd-trace:span:start')
+const { APM_TRACING_ENABLED_KEY, APPSEC_PROPAGATION_KEY } = require('../constants')
+
+let enabled
+
+function isStandaloneEnabled () {
+  return enabled
+}
+
+function onSpanStart ({ span, fields }) {
+  const { parent } = fields
+  const tags = span.context()._tags
```

Comment:
optional chaining everywhere on those, there is no garantee we have those objects, and we really don't want to crash

Reply:
I believe it is guaranteed that a span always has a context and a span context always has `_tags` property.
The code is full of expressions like `span.context()._tags[TEST_STATUS]` without the check.

But I have protected it...

Reply:
oh ok then, if you think it's not needed, i trust you